oauth2server/proj/settings/default.py

```diff
@@ -81,4 +81,128 @@
         'rest_framework.authentication.BasicAuthentication',
         'rest_framework.authentication.SessionAuthentication',
     )
-}
\ No newline at end of file
+}
+
+
+# Custom logging to give us unique logging to work on remote containers and normal logging
```

Comment:
Should the commented blocks be removed? Not sure what the policy is with commented out code blocks.